maplibre/src/context.rs

```diff
@@ -13,6 +13,8 @@ use crate::{
 /// Stores the camera configuration.
 pub struct ViewState {
     pub zoom: ChangeObserver<Zoom>,
+    pub maxZoom: u8,
+    pub minzoom: u8,
```

Comment:
Same goes for this attribute. Also both should be f64. Zoom is a float. A zoom level is u8.

Reply:
Found the u8 in style.rs and used it. But yes, vector renderers never should force a "jumping" zoom.